cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseStreamAdmin.java

```diff
@@ -52,9 +52,8 @@ public HBaseStreamAdmin(Configuration hConf, CConfiguration cConf, LocationFacto
   @Override
   public String getActualTableName(QueueName queueName) {
     if (queueName.isStream()) {
-      //TODO: namespace the tableName (have to modify tableNamePrefix)
-      // <cdap namespace>.system.stream.<stream name>
-      return getTableNamePrefix() + "." + queueName.getSecondComponent();
+      // <cdap namespace>.system.stream.<namespace>.<stream name>
+      return getTableNamePrefix() + "." + queueName.getFirstComponent() + "." + queueName.getSecondComponent();
```

Comment:
I also saw a similar table name generator method in ``StreamUtils``. Or was that for the state store? Worth consolidating?